### ISSUE TITLE
Assertion `cur_target >= 0 && cur_target < n_classes' failed.  

### ISSUE THREAD

**Issue by albertparlys** [amdegroot/deepgenres.torch#1]
Why this happen? Please help.

> /home/albert/torch/install/bin/luajit:` /home/albert/torch/install/share/lua/5.1/nn/THNN.lua:110: Assertion `cur_target >= 0 && cur_target < n_classes' failed.  at /tmp/luarocks_nn-scm-1-3484/nn/lib/THNN/generic/ClassNLLCriterion.c:48 stack traceback: 	[C]: in function 'v' 	/home/albert/torch/install/share/lua/5.1/nn/THNN.lua:110: in function 'ClassNLLCriterion_updateOutput' 	...ert/torch/install/share/lua/5.1/nn/ClassNLLCriterion.lua:41: in function 'forward' 	train.lua:69: in function 'opfunc' 	/home/albert/torch/install/share/lua/5.1/optim/rmsprop.lua:35: in function 'rmsprop' 	train.lua:76: in function 'train' 	train.lua:90: in main chunk 	[C]: in function 'dofile' 	...bert/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk 	[C]: at 0x00405d50`

Thank you :)

**Comment by amdegroot** [comment]
Were you trying to train on your own dataset? Or what was the issue?

**Comment by albertparlys** [comment]
Thank you for replying my issues.

Yes I trained my own dataset. I didn't create empty folders for raw,
spectograms, model, logs, etc. So the program didn't work. After I created
these empty folders, the program run properly.
But now I have a problem for testing my own pre trained model. The accuracy
of the genre is very low (about 25%). What did I do wrong?

Sorry for my bad English, I hope you understand what I mean.

Thank you.
